apps/dashboard/src/components/editor/use-autosave.js

```diff
@@ -19,8 +19,9 @@ export const useAutosave = ( projectId, editorView ) => {
 
 	const autosave = useRef();
 
-	const [ isEditorContentSaved ] = useSelect( ( select ) => [
```

Comment:
If the file isn't used, let's either remove it or at least remove those changes from the diff.